packages/system/kubeovn/charts/kube-ovn/templates/ovncni-ds.yaml

```
@@ -29,16 +29,52 @@ spec:
       hostNetwork: true
       hostPID: true
       initContainers:
+      - name: hostpath-init
```

yamllint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)